discovery/README.md

```diff
@@ -10,8 +10,18 @@ Docker Swarm comes with multiple Discovery backends.
 
 ## Backends
 
+You use a hosted discovery service with Docker Swarm. The service
+maintains a list of IPs in your swam. There are several commercial
```

Comment:
@thaJeztah crap! that is a good point. Sorry @MHBauer LGTM when this change is made. We will merge!